packages/terra-table/src/_variables.scss

```diff
@@ -1,55 +1,56 @@
 // Table Component Variables
 // ========================================================
-
-// Background Colors
-$terra-table-background-color: #fff;
-$terra-table-row-striped-background-color: #f4f4f4;
-$terra-table-subheader-background-color: #c8cacb;
-$terra-table-thead-background-color: #dedfe0;
-
-// Base Border Styles
-$terra-table-base-border-color: #c8cacb;
-$terra-table-base-border-width: 1px;
-
-// Borders
-$terra-table-border: 0;
-$terra-table-thead-tr-border-top: $terra-table-base-border-width solid $terra-table-base-border-color;
-$terra-table-thead-tr-border-bottom: 0;
-$terra-table-tfoot-tr-border-top: 0;
-$terra-table-tfoot-tr-border-bottom: $terra-table-base-border-width solid $terra-table-base-border-color;
-$terra-table-row-border-vertical: $terra-table-base-border-width solid $terra-table-base-border-color;
-$terra-table-row-border-horizontal: $terra-table-base-border-width solid $terra-table-base-border-color;
-$terra-table-cell-border: 0;
-
-// Table Header Font Styles
-$terra-table-thead-th-font-size: 1rem;
-$terra-table-thead-th-font-weight: bold;
-$terra-table-thead-th-color: #1c1f21;
-$terra-table-sort-indicator-color: #1c1f21;
-
-// Table Subheader Font Styles
-$terra-table-subheader-font-size: 1rem;
-$terra-table-subheader-font-weight: bold;
-$terra-table-subheader-color: #1c1f21;
-
-// Cell Padding
-$terra-table-cell-padding-bottom: 0.5em;
-$terra-table-cell-padding-left: 0.75em;
-$terra-table-cell-padding-right: 0.75em;
-$terra-table-cell-padding-top: 0.5em;
-
-// Table Cell Min Width
-$terra-table-cell-min-width-tiny: 3rem;
-$terra-table-cell-min-width-small: 4.25rem;
-$terra-table-cell-min-width-medium: 5.75rem;
-$terra-table-cell-min-width-large: 8.5rem;
-$terra-table-cell-min-width-huge: 10.5rem;
-
-// Selection Styles
-$terra-table-row-hover-background-color: #ebf6fd;
-$terra-table-row-selected-background-color: #ebf6fd;
-$terra-table-row-selected-hover-background-color: darken(#ebf6fd, 5%);
-$terra-table-row-selected-border-color: #0092e0;
-$terra-table-row-striped-hover-background-color: darken(#ebf6fd, 2%);
-$terra-table-row-striped-selected-background-color: darken(#ebf6fd, 5%);
-$terra-table-row-striped-selected-hover-background-color: darken(#ebf6fd, 10%);
+:root {
+  // Background Colors
+  --terra-table-background-color: #fff;
+  --terra-table-row-striped-background-color: #f4f4f4;
+  --terra-table-subheader-background-color: #c8cacb;
+  --terra-table-thead-background-color: #dedfe0;
+
+  // Base Border Styles
+  --terra-table-base-border-color: #c8cacb;
+  --terra-table-base-border-width: 1px;
+
+  // Borders
+  --terra-table-border: 0;
+  --terra-table-thead-tr-border-top: var(--terra-table-base-border-width) solid var(--terra-table-base-border-color);
```

Comment:
With the work we are planning to do in the [theme provider component](https://github.com/cerner/terra-core/tree/theme-adapter/packages/terra-theme-provider), CSS custom property values need to be static, rather than dynamic. With run-time theming, the plan is to find/replace CSS custom property with their value. If those values are dynamic e.g. (other CSS custom properties) that becomes a lot more challenging to polyfill in the run time environment. It will work with postcss-custom-properties plugin at build time though as the plugin can parse the AST and find the dynamic values.

tl:dr We should avoid using CSS custom properties as values to other CSS custom properties.

I'd recommend removing CSS custom properties like `--terra-table-thead-tr-border-top` and putting their value inline where they were used. 

e.g 
```css
 tr {
  border-bottom: var(--terra-table-base-border-width) solid var(--terra-table-base-border-color);
  border-top: var(--terra-table-base-border-width) solid var(--terra-table-base-border-color);
}
```

or make a variable for the entire short-hand value

```css
:root {
  --terra-table-thead-tr-border-bottom: 1px solid #c8cacb;
  --terra-table-thead-tr-border-top: 1px solid #c8cacb;
}
 tr {
  border-bottom: var(--terra-table-thead-tr-border-bottom);
  border-top: var(--terra-table-thead-tr-border-top);
}
```

I'll look into working on a stylelint rule for this case.